cadc-registry-server/src/main/java/org/opencadc/reg/oai/OAIWriter.java

```diff
@@ -181,6 +181,17 @@ private Document createEnvelopeOAI(URL oaiEndpoint, String oaiRequest) {
         return ret;
     }
 
+    private void recursiveSetNS(Element e, Namespace ns) {
```

Comment:
Hmm, not sure why there's a change in this class.  Perhaps your IDE is re-arranging content for you?

Reply:
The overloaded methods of addContent() were separated by this recursiveSetNS() method. So I moved it out of the way. This is to satisfy checkstyleMain.